tests/unit/latency-monitor.tcl

```diff
@@ -31,15 +34,15 @@ start_server {tags {"latency-monitor needs:latency"}} {
         assert_match {calls 1 histogram_usec *} [latency_histogram hset]
         assert_match {calls 1 histogram_usec *} [latency_histogram hgetall]
         assert_match {calls 1 histogram_usec *} [latency_histogram get]
-        assert {[llength [r latency histogram]] == 8}
+        assert {[llength [r latency histogram]] == 12}
```

Comment:
12? which are they?
maybe this test (which is about LTATENCY HISTOGRAM **with** argument, should not have this assert at all.

but instead, let's also add something like
```
        assert_match {config|resetstat*calls 1 histogram_usec *} [latency_histogram config|resetstat]
```
and maybe
```
        assert_match {config|resetstat*calls 1 histogram_usec *} [latency_histogram config]
```
the last one will not work with the current code, but in COMMAND INFO it will, so maybe we should change the code to support that?